fragmentCodeAndTransport.adoc

```diff
@@ -98,24 +98,29 @@ line 56. The least significant byte is output first, this means 32 is
 byte 0, 04 is byte 1 and and the final value 02 is byte 4.
 
 ==== Siemens transport
-
-The packet format is given in
-<<fig:packet-format>>. So this means the packet
-will be packed as follows:
+The packet is encapsulated according to the https://github.com/riscv-non-isa/e-trace-encap/releases/latest/[Unformatted Trace & Diagnostic Data Packet Encapsulation for RISC-V Specification], with the following attributes: 
 
 * Header - 1 byte
-* Index - N bits. As an example use 6 bits and the value of 1.
-* Optional Siemens timestamp - 2 bytes. This example has no timestamp
-* A type field for the packet of 2 bits ’01’ meaning instruction trace
-* Payload - [32 04 00 00 02]
+* SrcID - N bits. As an example use 6 bits and the value of 1.
+* This example has no timestamp
+* A 2-bit type field with ’10’ meaning instruction trace
+* trace_payload - [32 04 00 00 02]
 
 Since the Siemens transport is byte stream based the data seen will be:
 
-`[0x05][0x41][0x32 0x04 0x00 0x00 0x02]`
+`[0x06][0x81][0x32 0x04 0x00 0x00 0x02]`
 
 ==== ATB transport
 
-Assuming at 32 bit ATB transport results in the following ATB transfers
+The packet is encapsulated according to the https://github.com/riscv-non-isa/e-trace-encap/releases/latest/[Unformatted Trace & Diagnostic Data Packet Encapsulation for RISC-V Specification], with the following attributes:
+
+* Header - 1 byte
+* SrcID - 0 bits as the SrcID is carried via the ATID field.
+* This example has no timestamp
+* No type field (encoder has no data trace support)
+* trace_payload - [32 04 00 00 02]
+
+Assuming a 32 bit ATB transport results in the following ATB transfers
 
 `[ATID=1] [ATBYTES = 3] [ATDATA = 0x00043205]` +
 `[ATID=1] [ATBYTES = 1] [ATDATA = 0x00000200]`
```

Comment:
Does ATBYTES use -1 notation and padded to next 4 byte boundary? So even if ATBYTES indicates 1 (which I presume is 2 bytes) then ATDATA is still 4 bytes?

Reply:
Yes exactly.  This is how ATB indicates beats where only some of the data is valid.
(it doesn't have to pad to the next beat boundary - you could pack the next packet contiguously with this one and leave ATBYTES = 3, but I think that's beyond the scope of this example, which just concerns itself with a single packet)